templates/tfengine/recipes/cicd.hcl

```diff
@@ -42,7 +42,15 @@ schema = {
       }
     }
     cloud_source_repository = {
-      description          = "Config for Google Cloud Source Repository Cloud Build triggers."
+      description          = <<EOF
+        Config for Google Cloud Source Repository Cloud Build triggers.
+
+        IMPORTANT: Cloud Source Repositories does not support code review or
+        presubmit runs. If you set both plan and apply to run at the same time,
+        they will conflict when trying to acquire the lock and may error out.
+        To get around this, you should only auto-apply for non-prod envs,
```

Comment:
To get around this, for 'shared' and 'prod' environment, set 'apply' trigger to not 'run_on_push', and for other environments, do not specify the 'plan' trigger block and let 'apply' trigger to  'run_on_push'.